README.md

```diff
@@ -888,3 +813,444 @@ Optional.Some(3).tap(x  =>  console.log("Hello") ||  x  +  1);
 Optional.None().tap(x  =>  console.log("Test"));
 
 ```
+
+### `Result`
+
+A model for the result of a computation chain.
+
+#### Ok
+
+`Result` subtype for truthy values.
+
+#### Error
+
+`Result` subtype for errors and falsy values.
+
+#### Aborted
+
+Like `Error`, but unrecoverable.
+
+#### Pending
+
+`Result` subtype for asynchronous data-types.
+
+#### `Result` construction
+
+A `Result` can be constructed by directly calling the subtype that you'd like to use:
+
+```javascript
+const result1 = Result.Ok(3);
+const result2 = Result.Error({ message: 'Ooops' });
+const result3 = Result.Pending();
```

Comment:
This **should not** be used

Reply:
The truth of the matter is - the other are rarely used, but Pending is more like an implementation detail - maybe it should even be made private to the module and unexposed.